src/platform/src/test/java/org/geoserver/platform/resource/ResourceTheoryTest.java

```diff
@@ -486,10 +486,10 @@ public void theoryRecursiveDelete(String path) throws Exception {
     }
 
     @Theory
-    public void theoryRootSlashIsIgnored(String path) throws Exception {
-        final Resource res = getResource(path);
-        final Resource res2 = getResource("/" + path);
-        assertEquals(res, res2);
-        assertEquals(res.path(), res2.path());
+    public void theoryRootSlashIsAbsoluteOnLinux(String path) throws Exception {
+        if (!SystemUtils.IS_OS_WINDOWS) {
```

Comment:
I don't think this is what we want, just ignoring a test on Windows makes the test pass but will the weird behavior remain?

Reply:
I believe I setup a test that covers the windows behaviour also. Using the fact that java ``user.home`` will be located on a drive letter in order to have something to test against.

Reply:
Adjusted this test so it uses an absolute path on both linux and windows.